fuel-vm/src/storage/predicate.rs

```diff
@@ -63,246 +92,281 @@ impl From<StorageUnavailable> for RuntimeError<StorageUnavailable> {
     }
 }
 
-impl<Type: Mappable> StorageInspect<Type> for PredicateStorage {
-    type Error = StorageUnavailable;
+impl<Type, D> StorageInspect<Type> for PredicateStorage<D>
+where
+    Type: Mappable,
+    D: StorageRead<BlobData>,
+{
+    type Error = PredicateStorageError;
 
     fn get(
         &self,
         _key: &Type::Key,
-    ) -> Result<Option<Cow<'_, Type::OwnedValue>>, StorageUnavailable> {
-        Err(StorageUnavailable)
```

Comment:
Is `StorageUnavailable still used`? Its definition has not been removed

Reply:
Removed